cdc/processor/pipeline/sorter.go

```diff
@@ -365,8 +365,6 @@ func (n *sorterNode) handleRawEvent(ctx context.Context, event *model.Polymorphi
 				zap.Uint64("resolvedTs", resolvedTs),
 				zap.Uint64("oldResolvedTs", oldResolvedTs))
 		}
-		atomic.StoreUint64(&n.resolvedTs, rawKV.CRTs)
```

Comment:
after remove this line, when the sorter's resolve ts push forward ?

Reply:
line 361 `oldResolvedTs := atomic.SwapUint64(&n.resolvedTs, resolvedTs)`